packages/app/src/app/overmind/factories.ts

```diff
@@ -77,7 +77,7 @@ export const withOwnedSandbox = <T>(
 ): AsyncAction<T> => async (context, payload) => {
   const { state, actions } = context;
 
-  if (!state.editor.currentSandbox.owned) {
```

Comment:
Makes sense! Could you wrap this in a single `if` statement? `if (state.editor.currentSandbox` and then the other if statements. I think that's a bit cleaner and less duplication.